src/modality-specific-files/physiological-recordings.md

```diff
@@ -72,122 +194,585 @@ A guide for using macros can be found at
+-->
+{{ MACROS___make_filetree_example({
+  "sub-01": {
+    "func": {
+      "sub-01_task-visualSearch_bold.json": "",
+      "sub-01_task-visualSearch_bold.nii.gz": "",
+      "sub-01_task-visualSearch_events.json": "",
+      "sub-01_task-visualSearch_events.tsv": "",
+      "sub-01_task-visualSearch_recording-eye1_physio.json": "",
+      "sub-01_task-visualSearch_recording-eye1_physio.tsv.gz": "",
+      "sub-01_task-visualSearch_recording-eye1_physioevents.json": "",
+      "sub-01_task-visualSearch_recording-eye1_physioevents.tsv.gz": ""
+    },
+  },
+}) }}
+
+The above example is extended to a binocular eye-tracker producing three signals
+(left and right eyes, plus a *cyclopean* recording), as follows:
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+  "sub-01": {
+    "func": {
+      "sub-01_task-visualSearch_bold.json": "",
+      "sub-01_task-visualSearch_bold.nii.gz": "",
+      "sub-01_task-visualSearch_events.json": "",
+      "sub-01_task-visualSearch_events.tsv": "",
+      "sub-01_task-visualSearch_recording-eye1_physio.json": "",
+      "sub-01_task-visualSearch_recording-eye1_physio.tsv.gz": "",
+      "sub-01_task-visualSearch_recording-eye1_physioevents.json": "",
+      "sub-01_task-visualSearch_recording-eye1_physioevents.tsv.gz": "",
+      "sub-01_task-visualSearch_recording-eye2_physio.json": "",
+      "sub-01_task-visualSearch_recording-eye2_physio.tsv.gz": "",
+      "sub-01_task-visualSearch_recording-eye2_physioevents.json": "",
+      "sub-01_task-visualSearch_recording-eye2_physioevents.tsv.gz": "",
+      "sub-01_task-visualSearch_recording-eye3_physio.json": "",
+      "sub-01_task-visualSearch_recording-eye3_physio.tsv.gz": "",
+      "sub-01_task-visualSearch_recording-eye3_physioevents.json": "",
+      "sub-01_task-visualSearch_recording-eye3_physioevents.tsv.gz": ""
+    },
+  },
+}) }}
+
+Given the above example file structures, a corresponding
+`sub-01_task-visualSearch_recording-eye1_physio.json` sidecar
+could read:
+
+```JSON
+{
+    "DeviceSerialNumber": "17535483",
+    "Columns": ["timestamp", "x_coordinate", "y_coordinate", "pupil_size"],
+    "EnvironmentCoordinates": "top-left",
+    "Manufacturer": "SR-Research",
+    "ManufacturersModelName": "EYELINK II CL v4.56 Aug 18 2010",
+    "RecordedEye": "right",
+    "SampleCoordinateSystem": "gaze-on-screen",
+    "SampleCoordinateUnits": "pixel",
+    "SamplingFrequency": 1000,
+    "SoftwareVersion": "SREB1.10.1630 WIN32 LID:F2AE011 Mod:2017.04.21 15:19 CEST",
+    "ScreenAOIDefinition": [
+        "square",
+        [100, 150, 300, 350]
+    ],
+    "pupil_size": {
+        "Description": "Pupil area of the recorded eye as calculated by the eye-tracker in arbitrary units (see EyeLink's documentation for conversion).",
+        "Units": "a.u."
```

Comment:
this only describes pupil_size but the tsv content below has 4 columns, wouldn't it be better to have all 4 described?

I think that users can figure it out but I would prefer to avoid having users interprate the specification and be explicit about it.

Reply:
I put all descriptions to be written in the json file by the converter...I thought this was just an example to show that all descriptions should be present